common/http/src/batch.rs

```diff
@@ -263,6 +263,7 @@ mod tests {
         assert_eq!(input, lines);
     }
 
+    #[ignore]
     #[tokio::test(flavor = "multi_thread", worker_threads = 3)]
```

Comment:
Bumping the duration of this to something like 5000 makes it a lot less fragile.